purchases_ui_flutter/ios/Classes/PurchasesUiPaywallView.swift

```diff
@@ -0,0 +1,62 @@
+import Flutter
+import UIKit
+import PurchasesHybridCommonUI
+import RevenueCatUI
+
+class PurchasesUiPaywallViewFactory: NSObject, FlutterPlatformViewFactory {
+    private var messenger: FlutterBinaryMessenger
+
+    init(messenger: FlutterBinaryMessenger) {
+        self.messenger = messenger
+        super.init()
+    }
+
+    func create(
+        withFrame frame: CGRect,
+        viewIdentifier viewId: Int64,
+        arguments args: Any?
+    ) -> FlutterPlatformView {
+        return PurchasesUiPaywallView(
+            frame: frame,
+            viewIdentifier: viewId,
+            arguments: args,
+            binaryMessenger: messenger)
+    }
+
+    /// Implementing this method is only necessary when the `arguments` in `createWithFrame` is not `nil`.
+    public func createArgsCodec() -> FlutterMessageCodec & NSObjectProtocol {
+          return FlutterStandardMessageCodec.sharedInstance()
+    }
+}
+
+class PurchasesUiPaywallView: NSObject, FlutterPlatformView {
+    private var _view: UIView
```

Comment:
This can be immutable. Also underscores aren't the Swift convention:
```suggestion
    private let view: UIView
```